src/prepack-cli.js

```diff
@@ -304,7 +304,7 @@ function run(
             console.error(`Unsupported debugDiagnosticSeverity: ${arg}`);
             process.exit(1);
           }
-          debuggerConfigArgs.diagnosticSeverity = arg;
+          debuggerConfigArgs.diagnosticSeverity = (arg: any);
```

Comment:
We frown upon type casts like this because they are checked in any way. Right now it is correct because of the condition in line 303, but in the future things might change. Perhaps not here, but it is better to just uniformly enforce a rule about this.

Reply:
I restored the invariant method again. 😄